src/models.ts

```diff
@@ -139,6 +151,11 @@ export interface IFilterMeasureTarget extends IBaseFilterTarget {
 
 export declare type IFilterTarget = (IFilterColumnTarget | IFilterHierarchyTarget | IFilterMeasureTarget);
 
+export interface IFiltersContainer {
+  target?: ITarget,
```

Comment:
Reading the comment from other PR:

in the product filters have levels - page level filter, report level filter and visual level filters. So I think **level** would be a bit more appropriate than scope. At the same time just ILevel instead of ITarget sounds strange. 

Maybe we should rename as follow: ITarget -> ILevelFilter, IPageTarget->IPageLevelFilter, IVisualTarget->IVisualLevelFilter.